CHANGELOG.md

```diff
@@ -6,10 +6,62 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Changed
+## [1.2.0-rc.2] - 2024-06-14
+
+> TRUE or FALSE, it's a problem!
 
+### Important Notes
  * libcontainer/cgroups users who want to manage cgroup devices need to explicitly
    import libcontainer/cgroups/devices. (#3452, #4248)
+ * If building with Go 1.22.x, make sure to use 1.22.4 or a later version.
+   (see #4233 for more details)
+
+### Added
+
+ * CI: add actuated-arm64. (#4142)
+
+### Fixed
+
+ * Set the default value of CpuBurst to nil instead of 0. (#4210, #4211)
+ * runc exec: fix setting rlimit_nofile. (#4195, #4265, #4282, #4290)
+ * libct/cg/fs: fix setting rt_period vs rt_runtime. (#4094, #4258)
+ * script/*: fix gpg usage wrt keyboxd. (#4189)
+ * Fix a debug log message for user namespaces in nsexec. (#4311)
+ * libct/cg: write unified resources line by line. (#4186)
+ * libct.Start: fix locking, do not allow a second container init. (#4271)
+ * Fix tests in debian testing (mount_sshfs.bats) - minimal fix. (#4245)
+ * Fix checkpoint/restore tests on actuated-arm64. (#4276)
+ * Fix codespell warnings. (#4291)
+ * libct/cg/dev: fix TestSetV1Allow panic. (#4295)
+ * tests/int/scheduler: require smp. (#4298)
+ * tests/int: swap-related fixes. (#4188)
+
+### Changed
+
+ * libcontainer: allow containers to make apps think fips is enabled/disabled for testing. (#4246)
+ * Set temporary single CPU affinity before cgroup cpuset transition. (#3923)
```

Comment:
this was reverted in #4283 and should **not** be released if the strategy is changing. It would be good to confirm here, @kolyshkin.

Reply:
Grr, yes, this was noted before and asked to remove it completely. Thanks for catching it, github has not the best interface to see if comments where addressed.

@lifubang can you remove it, please? :)

Reply:
Thanks, I don't know why I didn't remove this. I think I have removed the both lines suggested by @kolyshkin in https://github.com/opencontainers/runc/pull/4317#discussion_r1634205864 
But...